appmonitoring/helm/appmonitoring-cr.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: azmon.app.monitoring/v1
```

Comment:
Let's not call the folder simply "helm". That suggests that it's used in a helm chart of the addon. Let's call it something that clearly indicates it's only used in the validation pipeline, not actual production code.

Reply:
done